packages/plugins/analytics/src/index.ts

```diff
@@ -0,0 +1,69 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the MIT License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019-Present Datadog, Inc.
+
+import { doRequest } from '@dd/core/helpers';
+import type { GlobalContext, PluginOptions } from '@dd/core/types';
+
+import { INTAKE_HOST, INTAKE_PATH, PLUGIN_NAME } from './constants';
+
+export { PLUGIN_NAME } from './constants';
+
+export const getAnalyticsPlugins = (context: GlobalContext): PluginOptions[] => {
+    const log = context.getLogger(PLUGIN_NAME);
+
+    context.sendLog = async (message: string, ctx: any = {}) => {
+        // Only send logs in production.
+        if (context.env !== 'production') {
+            return;
+        }
+
+        try {
+            const bundler = {
+                name: context.bundler.name,
+                version: context.bundler.version,
+            };
+
+            await doRequest({
+                // Don't delay the build too much on error.
+                retries: 2,
+                minTimeout: 100,
+                url: `https://${INTAKE_HOST}/${INTAKE_PATH}`,
+                method: 'POST',
+                type: 'json',
+                getData: async () => {
+                    const data = {
+                        ddsource: `@datadog/${bundler.name}-plugin`,
+                        env: context.env,
+                        message,
+                        service: 'build-plugins',
+                        bundler,
+                        plugins: context.pluginNames,
+                        version: context.version,
+                        team: 'yoann',
+                        ...ctx,
```

Comment:
FYI: if `ctx` contains `service`, or `bundler`, or `plugins`, it'll override the fields you setup here. Idk if it's expected or not, but if it isn't, you may want to put `...ctx` as the 1st element of this object

Reply:
I'm not sure what's preferable, being able to override the defaults, or losing what you may pass as the context.

I'll rename it as `overrides` so it's clearer that you can bypass the defaults with it.